clients/fides-js/src/lib/consent-types.ts

```diff
@@ -40,17 +40,10 @@ export type FidesOptions = {
 export class SaveConsentPreference {
   consentPreference: UserConsentPreference;
 
-  noticeHistoryId: string;
```

Comment:
simplified this to take the whole `notice` object, since we need it in order to access the cookie object in `preferences.ts`